client/assets/js/videoplayer.js

```diff
@@ -0,0 +1,22 @@
+angular.element(document).ready(function () {
```

Comment:
I add script on Welcome-page <script src="assets/js/videoplayer.js"></script> or you meen I have to write here all code?